core/src/main/java/com/distkv/core/DistkvNavigableMap.java

```diff
@@ -0,0 +1,23 @@
+package com.distkv.core;
+
+import static java.util.Map.Entry;
+
+public interface DistkvNavigableMap<K, V> extends DistkvMapInterface<K, V> {
```

Comment:
Add comment document for this interface.

And also add documents for its methods below.